index.html

```diff
@@ -1,31 +1,201 @@
+            </div>
+            <a href=""><button class="oversight">See-Project</button></a>
+        </div>
+        <div class="container">
+            <div class="container1">
+                <h2 class="cont-header">Professional Art Printing Data</h2>
+                <p class="cont-text"> A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                    has been the industry's standard
+                </p>
+                <ul class="lang">
+                    <button><li class="lingos" id="btn2">html</li></button>
+                    <button><li class="lingos" id="btn2">css</li></button>
+                    <button><li class="lingos" id="btn2">javascript</li></button>
+                </ul>
+            </div>
+            <a href=""><button class="oversight">See-Project</button></a>
+        </div>
+        <div class="container">
+            <div class="container1">
+                <h2 class="cont-header">Professional Art Printing Data</h2>
+                <p class="cont-text"> A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                    has been the industry's standard
+                </p>
+                <ul class="lang">
+                    <button><li class="lingos" id="btn2">html</li></button>
+                    <button><li class="lingos" id="btn2">css</li></button>
+                    <button><li class="lingos" id="btn2">javascript</li></button>
+                </ul>
+            </div>
+            <a href=""><button class="oversight">See-Project</button></a> 
+        </div>
+        <div class="container">
+            <div class="container1">
+                <h2 class="cont-header">Professional Art Printing Data</h2>
+                <p class="cont-text"> A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                    has been the industry's standard
+                </p>
+                <ul class="lang">
+                    <button><li class="lingos" id="btn2">html</li></button>
+                    <button><li class="lingos" id="btn2">css</li></button>
+                    <button><li class="lingos" id="btn2">javascript</li></button>
+                </ul>
+            </div>
+            <a href=""><button class="oversight">See-Project</button></a>
+        </div>
+    </section>
+    
+    <section class="About-me">
+        <div class="picfix">
+        <div class="abt">
+            <h3 class="bou-me">About me</h3>
+            <p class="abt-text">Hello I’m a software developer!
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+           <a href=""> <button class="resume">Get My Resume</button></a>
```

Comment:
- [ ] As the previous reviewer mentioned kindly make sure all `See Project` buttons follow the button styling: Active, Hover, and Pressed (see design) as from Figma design, for now, it doesn't change the color back when active
- [ ] [OPTIONAL] Kindly only use a button tag or a tag for this link/button